index.html

```diff
@@ -15,10 +15,12 @@
     <main>
       <header>
         <section>
```

Comment:
Does your `h1` really need to be in its own section? Semantically, isn't `h1` already a header tag?